pages/index.tsx

```diff
@@ -123,10 +124,13 @@ export const getStaticProps: GetStaticProps<HomeProps> = async () => {
   });
   const recentEvents = eventResponse.data;
 
+  const projects = await getProjects();
```

Comment:
Could we make this a hook instead of calling `await` inside the function?

(in the future, this is a great candidate for [React Suspense](https://reactjs.org/docs/concurrent-mode-suspense.html)!)

Reply:
So instead of passing the projects list down as static props at build time, we would want to have the octokit request be done client-side within a react hook? Since I don't think we can call react hooks inside getStaticProps.

Reply:
Oh shoot, that's a great point - totally slipped my mind. No, let's keep this current behaviour, sorry for the confusing comment!

(I guess my concern here more is how we can split up this request / make this more performant, since first load is a bit slow)